docs(rfc): RFC-0003 Rename the `api` app (#2453)
neicnordic/sensitive-data-archive: docs/rfcs/0003-rename-api-app.md

---
status: exploring
date: "2026-05-28"
authors:
   - "@jbygdell"
consulted: []
informed: []
---

> # Rename the APi app

Reviewer: ```suggestion
# Rename the API app
```

## Context and Problem Statement

> Currently the management API is locates unser `sda/cmd/api` and when built the comntainer command becomes `sda-api`. Since it can perform administrative tasks it is easy to refer to it as `sda-admin`, somthing that conflicts with the CLI tool with the same name.

Reviewer: ```suggestion
Currently the management API is located under `sda/cmd/api` and when built the container command becomes `sda-api`. Since it can perform administrative tasks it is easy to refer to it as `sda-admin`, something that conflicts with the CLI tool with the same name.
```

## Decision Drivers

* **Consistency** - applications should have a name that is related to the tasks they perform

## Considered Options

> * **`admin-api`** - Not a good choise since through RBAC restrictions submitters can also use the app.

Reviewer: ```suggestion
* **`admin-api`** - Not a good choice since through RBAC restrictions submitters can also use the app.
```

> * **`manager` or `management`** - Posible new name that is better in describing the application.

Reviewer: ```suggestion
* **`manager` or `management`** - Possible new name that is better describes the application.
```

## Open Questions

> * Descide on renaming the api app

Reviewer: ```suggestion
* Decide on renaming the API app
```
